### DISCUSSION TITLE
HomeKit application defaults when adding new item - can they be changed?

### DISCUSSION THREAD

**Discussion by sahafeez** [kuestess/homebridge-platform-insteonlocal#227] [General]
So when I rename things, or add a bunch of new items in the InsteonLocal they show up in the home app with Favorites and Status set to YES and it a pain to go through 80+ items to turn that off. Anyone know of any special magic to change that default behavior, or application you can DL that will expose the right knobs?

**Comment by kuestess** [comment]
@sahafeez Good question, interested if anyone has an answer.  Pain in the butt to go through everything when you have a ton of devices.
